src/core/core.controller.js

```diff
@@ -322,10 +323,11 @@ module.exports = function(Chart) {
 			}
 
 			// Clip out the chart area so that anything outside does not draw. This is necessary for zoom and pan to function
-			this.chart.ctx.save();
-			this.chart.ctx.beginPath();
-			this.chart.ctx.rect(this.chartArea.left, this.chartArea.top, this.chartArea.right - this.chartArea.left, this.chartArea.bottom - this.chartArea.top);
-			this.chart.ctx.clip();
+			var context = this.chart.ctx;
```

Comment:
@simonbrunel I'm okay with it since the minifier will get it. In terms of making everything consistent (always using ctx) then I think that can be done later. We'll probably be back in these files soon to do more minification